packages/blockchain-extension/resources/tutorials/developer-tutorials/privateData.md

```diff
@@ -104,71 +104,72 @@ It is also worth having a look at the collections.json file at the root of the f
 
 In this step, we will take you through the steps of creating a 2 org network. However, if you already have a 2 org network set up, feel free to use that instead.
 
-> Note: If you use your own network, you will need to change the MSP ID in the collections.json file once the contract has been generated.
+> Note: If you use your own network, add it as a collection in the collections.json file once the contract has been generated.
 
 1. Hover over the `FABRIC ENVIRONMENTS` panel on the left hand side of the screen and select the +.
 2. You will then be asked to `Select a method to add an environment`. Choose the `Create new from template` option.
 3. From the list of network configuration options, choose the `2 Org template (2 CAs, 2 peers, 1 channel)`.
-4. Give your network a name e.g. `newNetwork`. Whatever you find easy to identify! Then press `enter` which will trigger your network runtime to start.
+4. Give your network a name e.g. `newNetwork`. Whatever you find easy to identify!
+5. Select `V2_0 (Recommended)` as the channel capability. Then press `enter` which will trigger your network runtime to start.
 
 After a few minutes, you will have your 2 org network ready to use your new Private Data Smart Contract on!
 
+This step also creates the implicit collections described in the previous step. As this is a 2 org network, 2 collections will be created. They will be created as `_implicit_org_Org1MSP` and `_implicit_org_Org2MSP` where `Org1MSP` and `Org2MSP` is the MSPID used to specify the organisation. Your MSPIDs for a network are found in the "Organizations" menu in the "Fabric Environments" panel.
```

Comment:
Can replace the first two sentences with:
`As implicit collections are a built-in Hyperledger Fabric feature, 2 implicit collections will be automatically created when you start the network. They will be created as ...`

Reply:
`where `Org1MSP` and `Org2MSP` are the MSPIDs used to identify the organisations`